test/grammar_test.rb

```diff
@@ -1,7 +1,7 @@
 # encoding: utf-8
 require_relative 'helper'
 
-class GrammarTest < Test::Unit::TestCase
+class GrammarTest < Minitest::Test
```

Comment:
I think this could be a good idea. However as currently i am just comparing the toml hash and json hash having complex toml files makes debuging hard.